packages/picker/test/index.ts

```diff
@@ -1412,6 +1412,41 @@ export function runPickerTests(): void {
                 'has focus ring?'
             ).to.be.false;
         });
+        it('does not close on document scroll', async () => {
+            const el = await fixture(html`
+                <div style="height: 200vh; padding: 50vh 0;">
+                    <sp-picker label="Select an option" placement="right">
+                        <sp-menu-item value="option-1">Option 1</sp-menu-item>
+                        <sp-menu-item value="option-2">Option 2</sp-menu-item>
+                        <sp-menu-item value="option-3">Option 3</sp-menu-item>
+                    </sp-picker>
+                </div>
+            `);
+
+            const picker = el.querySelector('sp-picker') as Picker;
+            await elementUpdated(picker);
+            await nextFrame();
+            await nextFrame();
+
+            expect(picker.open).to.be.false;
+
+            const opened = oneEvent(picker, 'sp-opened');
+            picker.click();
+            await opened;
+
+            expect(picker.open).to.be.true;
```

Comment:
Since we're not testing the picker opening action in this test, would it be beneficial  to set up the test with picker already set to open and in this test, just validate that it's in an open state before proceeding?

Reply:
No, it wouldn't be beneficial in this case. The test is specifically checking that the picker remains open after scrolling, so you need to first verify it opens correctly and then verify it stays open after the scroll event. Setting it to open programmatically would bypass testing the actual opening behavior.

Reply:
It seems to me you need to verify _that it is open_ not that open action can be performed via JS.